### ISSUE TITLE
VSP Roadshow deck

### ISSUE THREAD

**Issue by rroueche** [department-of-veterans-affairs/va.gov-team#3243]

**Comment by rroueche** [comment]
Audience: is this to pitch to ppl who don't know about the platform and say "here's what we do"? or for onboarding ppl to be explicit and detailed about how to work with us?

Not recruiting.

Use case: a team is about to start on the Platform and we need to set expectations w/the Stakeholders at VA.
- Instill confidence that we don't do 508 the way you do it, so it'll feel different, but this way is better 
- Here are the expectations we have for anyone building on Platform, you should be aware of these as the person who'll be directing your team(s)
- We're going to help you be successful at DevOps, Agile, HCD, and this is how so be ready!
- Build comfort/trust/rapport
- Setting ground rules

**Comment by rroueche** [comment]
What outcomes do ppl equate Agile with? OIT there might be a spectrum, FROM: Agile is required and I need to tell someone I've done it, TO: I want to be more efficient/effective at solving a business problem.

Outside of OIT people may not be getting those directives (Agile, DevOps, HCD, etc)...

Some more interested in abdicating involvement, more interested in who do I assign to make sure a thing is done. Who in my dept is on the software work, and what are they supposed to be telling me, and are they doing agile y/n? Potential message: "less last minute marathons bc of things going awry"

This is a challenge we will have to address: Agile mindset vs Deadline mindset. When can I tell my boss when a specific set of features is going to launch? If the larger business org equates devops with business transformation, or if they think it's squarely isolated within OIT.

**Comment by rroueche** [comment]
For the "explicit things we won't do": consider talking through a widely known within VA successful thing, and the things VSP does that made it successful.

Lense:
1. What is the touchpoint they know (what does the user see): I can go to va.gov and submit a claim and this is what it looks like.
2. What is the platform doing underneath that
3. Why that's good.

Add screenshot of a spinning thing on FE of VA.gov for the Incident Response screenshot.

**Comment by KevinMHoffmanUSDS** [comment]
Interviewed three different DEPO product leads before the holidaze, the results are in MURAL here, but I've also added text in this comment. 

MURAL Capture: https://app.mural.co/invitation/mural/departmentofveteransaffairs9999/1577112054308?sender=kevinhoffman16624&key=3bdbc34a-d9d2-4c3c-9bcc-a3a13a274e07

What are some of the specific benefits you've experienced by receiving support or working with the VA.gov platform team?

- Helps us to understand what guardrails are in place
- Onboard all of the new people
- Opportunties to collaborate across teams that the platform team facilitates/meetings (TOT)
- Resources and assets so that we aren't inventing the wheel
- Concept of the launch checklist, resources around that work
- Tools like Flipper, used by Auth Experience
- Resources: Design system, QA resources 
- Manage a centralized set of APIs that everyone can use
- Platform spreads the word regading Flipper
- Haven't used the platform until the past month or so
- Worry about system load and performance, availability, resiliance of it (the web service and all of its' parts)
- First "release" adding one word to improve a fix, building on what's there
- Access to resources (people and documentation) to determine LOE

What should a good platform provide to a product leader?

- Onboarding process is great, but it's drinking from a firehose
- Much more clear idea of what each team within platform does
- The model of the platform thought through for teams that aren't VSA (e.g. Booz)
- When you hit the problem, you don't quite remember who to talk to
- How do we communicate with each team in platform
- Gaps in non VSA teams process, that don't address how these teams work
- Slack is great, but there are too many channels, to talk to people across platform
- Procurement introduces new challenges to handle team/resource turnover
- Contextual sessions for developers/designers
- Suffered similar pains (see next note)
- Leadership that manages the comms flow throughout the platform team
- Take pieces of maintaining platform quality, put that on the team
- Having a PR in place, "I'm waiting on," 
- If I don't know the functions of platform teams, the VSA teams don't know
- To have a better understanding of what to expect when we make requests of the platform team
- Onboarding can be more spread out, example: platform in 3 3 hours meetings, don't have a computer yet, don't have access to the code
- How do we "unstick this," is it "stuck," what's the turnaround time, etc.? 
- Rules have become rigid for posting in Slack, but the teams don't understand what to respect (e.g. comms)
- Where is the line between "product" onboarding and "platform onboarding" 
- The platform is treating Slack like a ticketing queue, but isn't treating Slack like a queue
- Platform is the place to have a consolidated message about how to work on our products
- I can work with it if I know what to expect
- Platform could play a more substantial role in how a DEPO lead comes in
- Transparency about what to expect
- More guidance on "how to work," how it all fits together
- Where is the line between people building on the platform and the platform?
- Previous role: design system, guardrails were done by app. dev team
- When you bring in agile, but the team is saying "I'm waiting for," 
- 24 hours is reasonable, but it's got to be shipped, we are dependent on a team.
